packages/shr-expand/README.md

```diff
@@ -15,7 +15,7 @@ $ npm install
 
 # Running the Tests
 
-This project contains unit tests for testing the SHR text importer, JSON schema exporter, and FHIR StructureDefinition exporter.  To run the tests, execute the following command:
+This project contains unit tests for testing the SHR text importer and FHIR StructureDefinition exporter.  To run the tests, execute the following command:
```

Comment:
Oh wow.  This line of text comes from a long time ago, like the first time we had all the code together in a single repo.  It's not quite right, but... I won't hold up the PR for this little nugget.